GUI/MainRepo.cs

```diff
@@ -48,25 +54,8 @@ public void UpdateRepo()
 
         private void UpdateRepo(object sender, DoWorkEventArgs e)
         {
-            try
```

Comment:
We actually catch them, but we are presenting the user now with the whole WebException, which is a step backward.